sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -1784,4 +1966,75 @@ void shutdown() {
       }
     }
   }
+
+  private static class AvroDeSe {
```

Comment:
better call it `AvroDeSer` if it is a game of words around `AvroSerDe` or even better just call it `AvroSerDe`